src/functionalTest/resources/application.yaml

```diff
@@ -307,3 +307,8 @@ document:
 southampton:
   court:
     email-address: ${SOUTHAMPTON_COURT_EMAIL_ADDRESS:}
+
+postcodelookup:
+  api:
```

Comment:
not required and revert this

Reply:
Retaining this until we could find an alternative approach.